src/server/cloak-server.js

```diff
@@ -473,10 +473,24 @@ function sendChosenWordList(room, answers){
             }
         }
     }
-
+    room.data.roundResults = toSend;
     if(roomMembers.length === Object.keys(toSend).length){
```

Comment:
would assigning object.keys(toSend).length to an appropriately named variable suffice?